### ISSUE TITLE
[v4] strapi watch-admin results in an error

### ISSUE THREAD

**Issue by creazy231** [strapi/strapi#11138]
### Describe the bug

Using `yarn strapi watch-admin` results in an error and a black page.

### Steps to reproduce the behavior

1. Setup a fresh project using the quickstart
2. (optional) generate a new plugin using `yarn strapi generate`
3. run `yarn strapi watch-admin`
4. See the error

### Expected behavior

It should start the dev server + frontend.

### Screenshots

![image](https://user-images.githubusercontent.com/13473174/135322103-e1e2f7e7-ee54-4674-8481-05625096153c.png)

### System

- Node.js version: v14.17.2
- NPM version: 6.14.13
- Strapi version: 4.0.0-beta.0
- Database: SQLite
- Operating system: OS X

**Comment by creazy231** [comment]
Would suggest to adjust the description of the `yarn strapi --help` command for `watch-admin`

**Comment by derrickmehaffy** [comment]
> Would suggest to adjust the description of the `yarn strapi --help` command for `watch-admin`

Yup, as mentioned in Discord chat using `yarn develop --watch-admin` works fine but we should adjust the help description to clarify this
